scripts/serverless.js

```diff
@@ -558,6 +559,7 @@ const processSpanPromise = (async () => {
               configuration,
               commandUsage,
               variableSources: variableSourcesInConfig,
+              interactiveContext,
```

Comment:
Wouldn't it be cleaner to expose `serverless` instance on interactive CLI context, and pass it to `.generateTelemetryPayload` (having that we probably don't have to make any new changes in `generateTelemetryPayload`)?

Reply:
That's definitely an option and might be a better solution overall 👍